src/app/public/modules/date-range-picker/types/date-range-calculator.ts

```diff
@@ -23,17 +23,33 @@ import {
 } from './date-range';
 
 export class SkyDateRangeCalculator {
+
+  /**
+   * Indicates the text displayed within the calculator select menu.
+   */
   public readonly shortDescription: string;
+
+  /**
+   * Indicates the calculator type.
+   */
   public readonly type: SkyDateRangeCalculatorType;
 
   constructor(
+    /**
+     * Indicates the calculator id.
+     */
     public readonly calculatorId: SkyDateRangeCalculatorId,
     private config: SkyDateRangeCalculatorConfig
   ) {
     this.type = config.type;
     this.shortDescription = config.shortDescription;
   }
 
+  /**
+   * Gets the current value of the calculator.
+   * @param startDateInput Indicates the start date value.
```

Comment:
```suggestion
   * @param startDateInput The start date.
```
Tweaking to follow the pattern of the other param descriptions on this page.